src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/nuxeo-enhanced-viewer-installation.md

```diff
@@ -0,0 +1,254 @@
+
+### Previewer High Availability
+
+The Previewer can be deployed in an High Availability way, to do so you can choose between [Hazelcast](https://hazelcast.com/) or [MongoDB](https://www.mongodb.com/) to share the HTTP sessions and OAuth2 tokens between previewer instances.
+
+#### Hazelcast
+
+Hazelcast is the preferred way to bring High Availability to the Previewer as it doesn't require to deploy another software.
+
+To configure it in Kubernetes, you first need to deploy an [Headless Service](https://kubernetes.io/docs/concepts/services-networking/service/#headless-services), like below:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: nev-viewer-hazelcast-headless
+  labels:
+    app: viewer
+    release: nev # to replace by the release name when you have deployed the NEV stack
+    component: viewer
+spec:
+  type: ClusterIP
+  clusterIP: None
+  ports:
+    - name: tcp-hazelcast
+      port: 5701
+  selector:
+    app: viewer
+    release: nev # to replace by the release name when you have deployed the NEV stack
+    component: viewer
+```
+
+Then you need to configure on the Previewer the environment variables below:
+
+```yaml:
+ARENDERSRV_NUXEO_OAUTH2_STORAGE_BACKEND: hazelcast
+ARENDERSRV_NUXEO_OAUTH2_STORAGE_HAZELCAST_SERVICE_DNS: nev-viewer-hazelcast-headless
+```
+
+#### MongoDB
+
+To use MongoDB as High Availability backend, you need to configure on the Previewer the environment variables below:
+
+```yaml
+ARENDERSRV_NUXEO_OAUTH2_STORAGE_BACKEND: mongodb
+ARENDERSRV_NUXEO_OAUTH2_STORAGE_MONGODB_SERVER: mongodb://mongodb-host:27017
+```
+
+The MongoDB server setting supports the [connection string](https://docs.mongodb.com/manual/reference/connection-string/).
+
+The HTTP sessions and OAuth2 tokens will be stored in the `sessions` and `oauth2Authorized` collections of the `arender` database.
+
+## ARender Customization
+
+You can customize the ARender Previewer interface to fit specific UI and UX needs.
+
+Examples of possible configurations:
+
+- Add or remove buttons from the ARender interface,
+- Modify ARender behaviors on specific user actions (like entering a comment when the user clicks on "Enter"),
+- Reference a new theme (by creating your custom CSS file),
+- etc.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Nuxeo Annotations with ARender/arender-customized.png
+    name: arender-customized.png
+    addins#screenshot#up_to_date
+--}}
+![arender-customized.png](nx_asset://a26f7eaf-e268-443f-be33-2c88542e2a13 ?w=600,border=true)
+
+The UI customization is done through two files to put in the `/docker-entrypoint-init.d` directory inside the Previewer container.
+
+- `arender.css` to customize the style
+- `arender.properties` to customize the behavior
+
+Please follow the [ARender Configuration Guide](https://docs.arender.io/fr/configuration/web-ui/) for more information about available properties and style.
```

Comment:
Should rather link to the english doc: https://docs.arender.io/configuration/web-ui/